src/Resources/config/core.php

```diff
@@ -54,6 +53,7 @@
             ])
 
         ->alias(Pool::class, 'sonata.admin.pool')
+        ->public()
```

Comment:
This shouldn't be public, if it's for tests to pass, we should add a `NEXT_MAJOR` comment to remove it when tests are rewritten.

Reply:
I'll wait for https://github.com/sonata-project/SonataAdminBundle/pull/7117 then

Reply:
If there are too many conflicts, I could merge that one.